addons/crm/static/src/views/forecast_kanban/forecast_kanban_model.js

```diff
@@ -75,6 +75,7 @@ export class ForecastKanbanDynamicGroupList extends ForecastKanbanModel.DynamicG
         const result = await super.load(...arguments);
         const lastGroup = this.groups.filter((grp) => grp.value).slice(-1)[0];
         if (lastGroup) {
+            // Moment is depreciating. We use Luxon now
```

Comment:
I just moved the code honestly didn't bother converting

Reply:
Could you do it when you have time? I think it's quite simple. The luxon api is not much different. I just have a doubt about what exactly you want to do and I didn't want to break anything :)

Reply:
@abd-msyukyu-odoo is the original developer for the whole temporal period thingy, I think it's more appropriate if he takes care of it 😊, would it be possible for you @abd-msyukyu-odoo ?

Reply:
@FrancoisGe @Williambraecky 
Hello! Sure I'll make a task to use `luxon` instead of `moment` in the CRM forecast flow. I think I already considered using luxon at that time and decided against it but I can't remember why, I guess I'll know if I had a good reason during de conversion :sweat_smile: